virtual_rainforest/models/abiotic/abiotic_tools.py

```diff
@@ -65,6 +76,71 @@ def calc_specific_heat_air(
         molar_heat_capacity_air: molar heat capacity, [J mol-1 C-1]
 
     Returns:
-        specific heat of air at constant pressure (J mol-1 K-1)
+        specific heat of air at constant pressure, [J mol-1 K-1]
     """
-    return 2e-05 * temperature**2 + 0.0002 * temperature + molar_heat_capacity_air
+    return DataArray(
+        (2e-05 * temperature**2 + 0.0002 * temperature + molar_heat_capacity_air),
+        dims=temperature.dims,
+        coords=temperature.coords,
+        name="specific_heat_air",
+    )
+
+
+def calculate_latent_heat_vaporisation(temperature: DataArray) -> DataArray:
+    """Calculate latent heat of vaporisation.
+
+    Args:
+        temperature: temperature, [C]
+
+    Returns:
+        latent heat of vaporisation, [J kg-1]
+    """
+
+    latent_heat_vaporisation = DataArray(
+        (-42.575 * temperature + 44994),
+        dims=temperature.dims,
+        coords=temperature.coords,
+        name="latent_heat_vaporisation",
+    )
+    return latent_heat_vaporisation
+
+
+def calculate_aero_resistance(
+    wind_speed: DataArray,
+    heat_transfer_coefficient: float,
+) -> DataArray:
+    """Calculate aerodynamic resistance.
+
+    Args:
+        wind_speed: wind speed at each canopy height, [m s-1]
+        heat_transfer_coefficient: factor that describes heat capacity of medium
+
+    Returns:
+        aerodynamic resistance
+    """
+
+    return DataArray(
+        (heat_transfer_coefficient / np.sqrt(wind_speed)),
+        dims=wind_speed.dims,
+        coords=wind_speed.coords,
+        name="areo_resistance",
+    )
+
+
+def set_layers_function(canopy_layers: int, soil_layers: int) -> List[str]:
```

Comment:
This functionality should probably go somewhere in the `AbioticModel` to make share it is applied consistently across all functions.

Reply:
It should probably be defined in the config and turn into core axes! But, fine where it is for the moment.